### ISSUE TITLE
Add Windows PowerShell ISE support for NVDA

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2836]
**Reported by oaron on 2012-11-29 20:49**
In Windows 7, and Windows 8, the Windows PowerShell consol and editor are included. NVDA works properly with the consol application, but in the editor NVDA doesn't read back the typed characters.

**Comment by JulienCochuyt** [comment]
Hi,

Any update on this one?

I've tried re-enabling events nameChange and stateChange just for the WpfTextView controls in PowerShell ISE (disabled as of f1b50b83ee for #2759): The console is live again (new content announced) but still no review, no braille, no caret.
Almost everything else in the GUI is fine.
Only WpfTextView controls (console tab, scripts tabs) need to be fixed.
Focusing the Commands tab would benefit from a dedicated gesture.

**Comment by Adriani90** [comment]
cc: @leonardder

**Comment by Adriani90** [comment]
cc: @codeofdusk

**Comment by leonardder** [comment]
I think visual studio code with its powershell extension is a much better choice to do powerShell coding.

**Comment by Adriani90** [comment]
the support for Windows 7 ended and Windows 8 support is also not really given for a very long time from now. I am closing this issue. Please open a new issue with the most recent version of Powershell ISE under Windows 10 or Windows 11 in case this is not accessible to NVDA in the meantime.
